grip/github_renderer.py

```diff
@@ -2,18 +2,18 @@
 from flask import abort, json
 
 
-def render_content(text, gfm=False, context=None,
+def render_content(text, api_url, gfm=False, context=None,
                    username=None, password=None):
     """Renders the specified markup using the GitHub API."""
     if gfm:
-        url = 'https://api.github.com/markdown'
+        url = '%s/markdown' % api_url
```

Comment:
Since you're making edits, mind using the newer `'{}/markdown'.format(api_url)`?

Thanks!